src/ledger.test.js

```diff
@@ -0,0 +1,62 @@
+import describe from 'riteway';
+
+import { ethereum as $ } from './index';
+import { $$, addPercent, subtractPercent } from './ledger';
+
+describe('$$', async assert => {
```

Comment:
We might want to make these tests more thorough by adding an additional assert for each case.

Reply:
Yes. Later, maybe. =)